dhall/doctest/Main.hs

```diff
@@ -27,10 +27,17 @@ main = do
         Test.DocTest.doctest
             [ "-DWITH_HTTP"
             , "--fast"
+            , prefix </> "ghc-src"
+
+            -- Unfortunately we cannot target the entire @src@ directory.
+            -- The reason is that src/Dhall/Version.hs depends on
+            -- the generated Paths_dhall module which is "out-of-scope"
+            -- when running the testsuite with cabal v1-test.
+            -- Instead, we target a selection of modules whose combined module
+            -- dependency tree covers all modules that contain doctests.
+
+            -- , prefix </> "src"
             , "-i" <> (prefix </> "src")
-            , "-i" <> (prefix </> "ghc-src")
-            , prefix </> "src/Dhall.hs"
```

Comment:
Is the new version still testing the `Dhall` and `Dhall.Import` modules?

Reply:
Yes. `Dhall.Tutorial` depends on `Dhall`, which depends on `Dhall.Import`.

Reply:
But does testing `Dhall.Tutorial` guarantee that `Dhall` is also tested?

Reply:
That is my current understanding. At least this logic explains why `Dhall.Map` was being tested. @quasicomputational can probably clarify.

And it seems that I should then clarify the comment above.